packages/components/src/internal/components/domainproperties/TextChoiceOptions.tsx

```diff
@@ -13,19 +13,14 @@ import { DisableableButton } from '../buttons/DisableableButton';
 import { DisableableInput } from '../forms/DisableableInput';
 
 import { DOMAIN_VALIDATOR_TEXTCHOICE, MAX_VALID_TEXT_CHOICES } from './constants';
-import {
-    DEFAULT_TEXT_CHOICE_VALIDATOR,
-    DomainField,
-    isValidTextChoiceValue,
-    ITypeDependentProps,
-    PropertyValidator,
-} from './models';
+import { DEFAULT_TEXT_CHOICE_VALIDATOR, DomainField, ITypeDependentProps, PropertyValidator } from './models';
 import { SectionHeading } from './SectionHeading';
 import { DomainFieldLabel } from './DomainFieldLabel';
 
 import { TextChoiceAddValuesModal } from './TextChoiceAddValuesModal';
 import { createFormInputId, getTextChoiceInUseValues } from './actions';
 
+const VALUE_SEARCH_COUNT = 2;
```

Comment:
The name didn't make it obvious what this was for.  Maybe a comment?  Or update to `MIN_VALUES_FOR_SEARCH_COUNT` (yeah, probably too long).

Reply:
long constant names don't scare me. updated to `MIN_VALUES_FOR_SEARCH_COUNT`